openapi.yml

```diff
@@ -0,0 +1,506 @@
+  /workflow_templates:
+    get:
+      summary: Retrieve workflow template for given workflow
+      responses:
+        '200':
+          description: Workflow datastreams found
+          content:
+            application/json:
+              schema:
+                type: string
+  /workflow_templates/{workflow}:
+    get:
+      summary: Retrieve workflow template for given workflow
+      responses:
+        '200':
+          description: Workflow datastream found
+          content:
+            application/json:
+              schema:
+                type: string
+      parameters:
+        - name: workflow
+          in: path
+          required: true
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'
+  /workflow_archive:
+    get:
+      summary: Retrieve archived workflows
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: workflow
+          in: query
+          required: false
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'
+  /workflow_queue:
+    get:
+      summary: Retrieve the workflow queue
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: completed
+          in: query
+          required: false
+          example: dor:etdSubmitWF:catalog-status
+          schema:
+            type: string
+        - name: waiting
+          in: query
+          required: false
+          example: dor:etdSubmitWF:catalog-status
+          schema:
+            type: string
+  /workflow_queue/lane_ids:
+    get:
+      summary: Retrieve the workflow queue based on the provided lane_id
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: step
+          in: query
+          required: true
+          example: dor:etdSubmitWF:catalog-status
+          schema:
+            type: string
+  /workflow_queue/all_queued:
+    get:
+      summary: Retrieve the workflow queue for all queues
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: limit
+          in: query
+          required: false
+          example: '10'
+          schema:
+            type: string
+components:
+  schemas:
+    Druid:
+      description: Digital Repository Unique Identifier (bare or prefixed)
+      type: string
+      pattern: '^(druid:)?[b-df-hjkmnp-tv-z]{2}[0-9]{3}[b-df-hjkmnp-tv-z]{2}[0-9]{4}$'
```

Comment:
Does workflow-server-rails allow or use bare druids?

Reply:
I did not see any bare druids in the logs. Could have missed it, but I don't think so.